modules/cij/runner.py

```diff
@@ -769,50 +875,62 @@ def main(conf):
 
     trun: TestRun
     try:
-        trun = trun_setup(conf)         # Construct 'trun' from 'conf'
+        trun = trun_setup(conf)     # Construct 'trun' from 'conf'
     except InitializationError as ex:
         cij.err("main:FAILED to start testrun: %s" % ex)
 
     trun_to_file(trun)              # Persist trun
     trun_to_junitfile(trun)         # Persist as jUNIT XML
-    trun_emph(trun)                 # Print trun before run
 
     tr_err = 0
     tr_ent_err = trun_enter(trun)
-    for tsuite in (ts for ts in trun.testsuites if not tr_ent_err):
+    for tplan in (tp for tp in trun.testplans if not tr_ent_err):
+        tp_err = 0
+        tp_ent_err = tplan_enter(trun, tplan)
+
+        for tsuite in (ts for ts in tplan.testsuites if not tp_ent_err):
+            ts_err = 0
+            ts_ent_err = tsuite_enter(trun, tsuite)
 
-        ts_err = 0
-        ts_ent_err = tsuite_enter(trun, tsuite)
-        for tcase in (tc for tc in tsuite.testcases if not ts_ent_err):
+            for tcase in (tc for tc in tsuite.testcases if not ts_ent_err):
+                tc_err = 0
 
-            tc_err = 0
+                tcase_match = conf.get("TESTCASE_MATCH", None)
```

Comment:
`tcase_match` assignment can be moved to before the `tplan` loop

Reply:
true, but it is unrelated to the multi-testrun, adding to list of things to cleanup.